migrations/m150610_162817_oauth.php

```diff
@@ -22,6 +22,7 @@ public function safeUp()
                 'client_id' => Schema::TYPE_STRING . '(80) NOT NULL',
                 'client_secret' => Schema::TYPE_STRING . '(80) NOT NULL',
                 'redirect_uri' => Schema::TYPE_TEXT . ' NOT NULL',
+                'user_id' => Schema::TYPE_INTEGER . ' NOT NULL',
```

Comment:
http://stackoverflow.com/questions/7522831/what-is-the-purpose-of-the-implicit-grant-authorization-type-in-oauth-2
Here is description of how to use implicit grant.